cloud/aci-integration.md

```diff
@@ -99,16 +99,16 @@ There are two ways to use your new ACI context. You can use the `--context` flag
 docker --context myacicontext run -p 80:80 nginx
 ```
 
-Or, you can change context using `docker context use` to select the ACI context to be your focus for running Docker commands. For example, we can use the `docker context use` command to deploy an ngnix container:
+Or, you can change context using `docker context use` to select the ACI context to be your focus for running Docker commands. For example, we can use the `docker context use` command to deploy an Nginx container:
 
 ```console
 docker context use myacicontext
-docker run -p 80:80 nginx
+docker run -p 80:80 Nginx

```

Comment:
This leads to an error.
```
docker: invalid reference format: repository name must be lowercase.
```

Reply:
Good catch . Let me fix this.

Reply:
@StefanScherer PTAL - https://github.com/docker/docker.github.io/pull/12823

Reply:
oh, nice catch 👍